userspace/falco/grpc_context.h

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright (C) 2016-2019 The Falco Authors
+
+This file is part of falco.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <string>
+
+#ifdef GRPC_INCLUDE_IS_GRPCPP
+#include <grpcpp/grpcpp.h>
+#else
+#include <grpc++/grpc++.h>
+#endif
+
+const std::string meta_session = "session_id";
+const std::string meta_request = "request_id";
+
+class context
```

Comment:
If this new class is only for `gRPC` then when aren't we labeling it as such?

```
class grpc_context
```

Reply:
You're right Kris :)

But we'd like to use namespaces for this! WDYT?

Reply:
I love namespaces!

Reply:
Done!